ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/GIT_Source/tasks/main.yml

```diff
@@ -41,6 +41,25 @@
   when:
     - (git_installed.rc != 0 ) or (git_installed.rc == 0 and git_version.stdout is version_compare('2.15', operator='lt'))
     - ansible_distribution != "FreeBSD"
+    - ansible_distribution != "CentOS" or ansible_distribution_major_version | int != 6
```

Comment:
We should probably have a RHEL clause alongside CentOS here, although I appreciate it's now an unlikely use case.

Reply:
Added.